### ISSUE TITLE
CircleCI failing on "cp -R /opt/solr-4.3.1 $SOLR_HOME"

### ISSUE THREAD

**Issue by cclauss** [ckan/ckan#4335]
### CKAN Version if known (or site URL)
https://github.com/ckan/ckan

### Please describe the expected behavior
CircleCI tests pass

### Please describe the actual behavior
$ __cp -R /opt/solr-4.3.1 $SOLR_HOME__
```
cp: cannot stat ‘/opt/solr-4.3.1’: No such file or directory
cp -R /opt/solr-4.3.1 $SOLR_HOME returned exit code 1
Action failed: cp -R /opt/solr-4.3.1 $SOLR_HOME
```
### What steps can be taken to reproduce the issue?
Run CircleCI on the current master
* Happens on Circle CI v1 #4334
* Happens on Circle CI v2 #4333

**Comment by cclauss** [comment]
@deinok is Circle CI tests broken?  A PR that does not change any code causes Circle CI to fail.  Am I doing something wrong?

**Comment by amercader** [comment]
This is definitely something on the CircleCI. Builds on master seem to work though:

https://circleci.com/gh/ckan/ckan/tree/master

I'm not sure why #4334 failed though. Did you change the image? Maybe this can help:

https://discuss.circleci.com/t/missing-opt-solr-4-3-1-in-both-12-04-and-14-04/6021/2

In any case I don't think Solr is available in Circle 2.0 by default, so we'll need to install it manually anyway (or use a custom Docker image, I'm not sure how circle 2.0 works)

**Comment by cclauss** [comment]
Closing.  Should be solved via Solr running in Docker at #4394
